iml-gui/crate/src/components/mod.rs

```diff
@@ -1,3 +1,4 @@
+pub(crate) mod breadcrumbs;
```

Comment:
Since it is not using anything from Seed, I am not sure now it here is a good place for it.